CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
+## [1.5.1]
```

Comment:
[X.X.X] instead
version is determined from all new features / fix in a specific PR

Reply:
Done